stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-14.27 # Last 8.6.5
+resolver: nightly-2020-11-22
```

Comment:
Could you explain why did you change the development resolver?

Reply:
I reverted back those changes. Checked in the wrong file.